.ci.yaml

```diff
@@ -1567,6 +1567,25 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Linux_android hello_world_impeller
+    recipe: devicelab/devicelab_drone
+    presubmit: true
+    bringup: true
+    timeout: 60
+    dimensions:
+      kvm: "1"
+      cores: "8"
+      machine_type: "n1-standard-8"
+    properties:
+      device_type: "none"
+      tags: >
+        ["devicelab", "linux"]
+      task_name: hello_world_impeller
+      dependencies: >-
+        [
+          {"dependency": "android_virtual_device", "version": "34"}
```

Comment:
I'm not sure if we are guaranteed an arm64 device which we need for this test.

Reply:
Is this supposed to run on an emulator?

Reply:
I don't know of any reason why it shouldn't work on an arm64 emulator.  We aren't changing the behavior of the renderer based on the capabilities of the device or anything, today at least.

Reply:
I mean, is the desire to run this on hardware? Where do you want this to run?

Reply:
It's probably safer to not run on the emulator for now.  I turned that off and also switched this to a post build task since this is testing engine and framework integration, no need to gate framework prs on this.

Reply:
> Where do you want this to run?

I'm ambivalent, either works.  I would suspect the emulator is easier on resources since it isn't gated on having a physical device.  I moved it over since that way I could be reasonably sure we'd get arm64.